pkgs/development/tools/scss_lint/default.nix

```
@@ -0,0 +1,15 @@
+{ lib, bundlerApp }:
+
+bundlerApp {
+  pname = "scss_lint";
```

```suggestion
  pname = "scss-lint";
```

because https://nixos.org/nixpkgs/manual/#sec-package-naming and it's also the upstream name
> The hyphenated style is preferred in all three package names.

You would need to rename the directory too.

---

:+1: I'll update.